### PR TITLE
basic website copy updates

### PR DESCRIPTION
**Pull request by brianwcook**

### CODE REVIEW

File: website/data/carousel/konflux.yaml [hunk 1]
```diff
@@ -1,10 +1,8 @@
 weight: 1
-title: "Welcome to Konflux"
+title: "Secure Builds Made Easy"
 description: >
   <div>
-    <p>Opinionated Kubernetes-native security-first software factory based on Tekton.<br>
-       Your one-stop shop for building, testing, and deploying source code with secure CI/CD.
-    </p>
+    <p>Konflux-ci is an open source, cloud native software factory focused on software supply chain security.    </p>
```
Comment (yftacherzog) [inline website/data/carousel/konflux.yaml]:
```suggestion
    <p>Konflux-ci is an open source, cloud-native software factory focused on software supply chain security.</p>
```

File: website/data/features/multicloud.yaml
Comment (yftacherzog) [inline website/data/features/multicloud.yaml:1]: With one removed, we have 5 which don't look as good I think. If possible, is there a feature we can add instead?
image

File: website/hugo.toml [hunk 2]
```diff
@@ -28,7 +28,7 @@ pluralizelisttitles = false
 [[menu.main]]
     name       = "User Guide"
     identifier = "menu.userguide"
-    url        = "https://redhat-appstudio.github.io/docs.appstudio.io"
+    url        = "https://konflux-ci.dev/docs"
```
Comment (yftacherzog) [inline website/hugo.toml:31]: That docs webpage seems to only generate titles at the moment. Do we prefer that for now over having the previous one in which the pages also had content?

Comment (brianwcook) [reply website/hugo.toml:31]: no, i don't want to link to app studio docs. I'd rather put pressure on populating konflux docs area.

File: website/data/features/multicloud.yaml
Comment (brianwcook) [reply website/data/features/multicloud.yaml:1]: done

**Review by ralphbean — APPROVED** [review]

**Pull request merged**